app/src/main/assets/html/error/unable-to-connect.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang="en">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Unable to Connect</title>
+    <body>
+        <script src="js/error.js"></script>
+        <div class="container">
+            <img class="icon" src="images/icon-errorpage-unabletoconnect.svg" alt="Unable to Connect">
+
+            <h1>Unable to Connect</h1>
```

Comment:
also, the `Connect` isn't capitalised in release-channel Firefox:

![image](https://user-images.githubusercontent.com/203725/42784866-560d90de-8905-11e8-87c8-397b42eff100.png)